src/bls/bls_worker.h

```diff
@@ -69,36 +69,36 @@ class CBLSWorker
     //   [ a1+a2+a3+a4, b1+b2+b3+b4, c1+c2+c3+c4, d1+d2+d3+d4]
     // Multiple things can be parallelized here. For example, all 4 entries in the result vector can be calculated in parallel
     // Also, each individual vector can be split into multiple batches and aggregating the batches can also be parallelized.
-    void AsyncBuildQuorumVerificationVector(const std::vector<BLSVerificationVectorPtr>& vvecs,
+    void AsyncBuildQuorumVerificationVector(Span<BLSVerificationVectorPtr> vvecs,
                                             size_t start, size_t count, bool parallel,
                                             std::function<void(const BLSVerificationVectorPtr&)> doneCallback);
-    std::future<BLSVerificationVectorPtr> AsyncBuildQuorumVerificationVector(const std::vector<BLSVerificationVectorPtr>& vvecs,
+    std::future<BLSVerificationVectorPtr> AsyncBuildQuorumVerificationVector(Span<BLSVerificationVectorPtr> vvecs,
```

Comment:
start/count can be refactored out, thanks to Span: [ac4f68d9fdac35068735bd896619475cbc2d4799](https://github.com/dashpay/dash/pull/5599/commits/ac4f68d9fdac35068735bd896619475cbc2d4799)